app/models/document.rb

```diff
@@ -64,7 +64,7 @@ def tag_metadata
 
   def tag_labels_for(key)
     Array(attrs.fetch(key, []))
-      .map { |tag| tag.fetch("label") }
+      .map(&method(:get_metadata_label))
```

Comment:
Given this is causing some confusion is it worth expanding this out into the more explicit `{ |tag| get_metadata_link(tag) }` style?